evergreen.yaml

```diff
@@ -23,18 +24,33 @@ variables:
 #              Functions              #
 #######################################
 functions:
+  git-config-windows-api-length:
+    command: shell.exec
+    type: setup
+    params:
+      silent: true
+      script: |
+        if [ "Windows_NT" == "$OS" ]; then
+          git config --system core.longpaths true
+        fi
   setup-credentials:
     command: shell.exec
     type: setup
     params:
       silent: true
       script: |
-        mkdir ~/.aws
-        cat <<EOF > ~/.aws/config
+        if [ "Windows_NT" == "$OS" ]; then
+          export AWS_DIR=/cygdrive/c/Users/$USER/.aws
+        else
+          export AWS_DIR=$HOME/.aws
+        fi
```

Comment:
push to makefile?

Reply:
don't want people to wipe out their aws directories by accident